src/app/auth/features/user-manager-detail/user-manager-detail.component.spec.ts

```diff
@@ -72,18 +111,18 @@ class MockActivatedRoute {
  */
 class MockRouter {
```

Comment:
MockRouter and  MockActivatedRoute is not needed when using the HttpTestingController (see below)